AccountantApp/src/main/java/com/tap/project/escrivaghera/AccountantApp/IllegalJournalEntryException.java

```diff
@@ -0,0 +1,9 @@
+package com.tap.project.escrivaghera.AccountantApp;
+
+public class IllegalJournalEntryException extends Exception {
+
+	public IllegalJournalEntryException(String error) {
```

Comment:
The constructor must be empty, create a final static field with following declaration: "The total of counts is several"